shopfloor/actions/message.py

```diff
@@ -249,6 +249,15 @@ def x_units_put_in_package(self, qty, product, package):
             ),
         }
 
+    def x_units_put_in_location(self, qty, product, location):
+        # TODO refactor with x_units_put_in_package
```

Comment:
maybe just `x_units_put_in(self, qty, product, destination)` and just pick the name of destination.